test/functional/rpc_invalidateblock.py

```diff
@@ -83,6 +84,9 @@ def run_test(self):
         # Should be back at the tip by now
         assert_equal(self.nodes[1].getbestblockhash(), blocks[-1])
 
+        self.log.info("Verify that invalidating an unknown block throws an error")
+        assert_raises_rpc_error(-5, "Block not found", self.nodes[1].invalidateblock, "00" * 32)
+
```

Comment:
might as well check it didn't effect chaintips, the important thing  :)

Reply:
Perfect, thanks!